transformations/formal2casual/README.md

```diff
@@ -0,0 +1,15 @@
+# Formal to Casual 🐍 → 🦎
+This transformation transfers text style from formal to casual (informal).
+
+Author Name: Rishabh Gupta (rishabh19089@iiitd.ac.in)
+
+## What type of a transformation is this?
+This transformation acts like a paraphraser by performing style transfer. It changes the tone of the text from formal to informal.
+Example: This car looks fascinating --> This car is so cool
+
+## What tasks does it intend to benefit?
+This perturbation would benefit all tasks which have a sentence/paragraph/document as input like text classification, 
+text generation, etc. 
+
```

Comment:
Please add a data and model section too like this: https://github.com/GEM-benchmark/NL-Augmenter/pull/116/files